### PR TITLE
doc: add release issue template

### PR DESCRIPTION
**Pull request by ret2libc**
# SQUASH ME

### CODE REVIEW

File: doc/release-template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] Ensure that CI on `stable` branch is all green. If not, fix issues and iterate previous steps as necessary.
+- [ ] Merge PR so that `origin/stable` is updated with the new content.
+- [ ] Manually test that things work well, in particular major changes in the new release.
+- [ ] Start preparing release notes by modifying the draft release automatically created by GitHub CI.
+- [ ] Check that [Cutter](https://github.com/rizinorg/cutter) works well with the new version of Rizin. If not, fix as necessary and go back to previous steps until everything is ready. Cutter and Rizin should be released at the same time, so both should be in a good state with major issues solved.
+  - [ ] Cutter can be compiled with new version of Rizin.
+  - [ ] Basic Cutter functions work well with new version of Rizin.
+  - [ ] Cutter is in good shape to be released.
+- [ ] Check that [rz-ghidra](https://github.com/rizinorg/rz-ghidra) works well with the new version of Rizin. If not, fix as necessary and go back to previous steps until everything is ready.
```
Comment (XVilka) [inline doc/release-template.md:19]: Add `jsdec` and `rz-retdec`.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: doc/release-template.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] Check that [rizin-extras](https://github.com/rizinorg/rizin-extras/) and rz-keystone in particular works well with the new version of Rizin. If not, fix as necessary and go back to previous steps until everything is ready.
+- [ ] Check that core plugins in [rz-pm-db](https://github.com/rizinorg/rz-pm-db) can be installed correctly with the new version of Rizin. If not, fix as necessary and go back to previous steps until everything is ready.
+- [ ] Replace `RZ_PIPE_PY_VERSION` and `RZ_GHIDRA_VERSION` in [`Dockerfile`](https://github.com/rizinorg/rizin/blob/dev/Dockerfile) with the right commits from those repositories. Merge the commit with the Rizin changes in the `stable` branch.
+- [ ] Quick test again that Rizin and Cutter work together.
+- [ ] Finalize release notes.
+- [ ] Release Rizin and Cutter by making their release public on GitHub.
+- [ ] Tag rz-pipe for the new version.
+- [ ] Tag rz-bindings for the new version.
```
Comment (XVilka) [inline doc/release-template.md]: `rz-bindgen`?

Comment (wargio) [reply doc/release-template.md]: the api needs to be updated.

Comment (ret2libc) [reply doc/release-template.md]: yes rz-bindgen. Shall I `s/rz-bindings/rz-bindgen`? However, it's the rz-bindings that really matter to users, not how they are generated from the rz-bindgen.

**Review by XVilka — APPROVED** [review]

**Pull request merged**